js/controller.js

```diff
@@ -115,6 +117,7 @@ if(playerForm !== null){
   playerForm.addEventListener('submit',handleSubmit);
 }
 if(playerHand !== null){
+  // todo: this function does not exist
   secondLoad();
```

Comment:
Might be a function you are planning to add in the future? If so, add a 'todo: implement' comment! This will clear up any confusion someone else might have.